lib/matplotlib/animation.py

```diff
@@ -1392,26 +1391,20 @@ class TimedAnimation(Animation):
 
     Parameters
     ----------
-    fig : `~matplotlib.figure.Figure`
-       The figure object that is used to get draw, resize, and any
-       other needed events.
-
-    interval : number, optional
-       Delay between frames in milliseconds.  Defaults to 200.
-
-    repeat_delay : number, optional
+    fig : `~.Figure`
+        The figure object that is used to get draw, resize, and any
+        other needed events.
+    interval : int, default: 200
+        Delay between frames in milliseconds.
+    repeat_delay : int, default: None
```

Comment:
```suggestion
    repeat_delay : int, default: 0
```
That's what it semantically does. AFAICS we could even use 0 as the actual default value.

Reply:
agreed.